pages/common/curl.md

```diff
@@ -25,4 +25,4 @@
 
 - pass a user name and password for server authentication
 
-`curl -u myusername:mypassword {{http://localhost}}`
+`curl -u {{username:password}} {{http://localhost}}`
```

Comment:
For the sake of consistency, I think this should be

`{{username}}:{{password}}`

We don't have a clearly defined syntax for what happens within the braces, and some people might not understand what the colon is doing there